demos/service-workflow/src/clients/farewell.ts

```diff
@@ -1,16 +1,16 @@
 import { Client } from '@temporalio/client';
+import { randomUUID } from 'node:crypto';
 import { farewell } from '../workflows';
-import { uuid4 } from '@temporalio/workflow';
 
 async function run() {
   const client = new Client();
-  const handle = await client.workflow.start(farewell, {
+  const result = await client.workflow.execute(farewell, {
```

Comment:
just my stylistic pref, if the only thing that's done with the handle is awaiting the result